features/sd-cmd.feature

```diff
@@ -0,0 +1,51 @@
+@ignore
+Feature: Commands
+
+    Users want to share binaries or scripts across multiple containers
+    so that they can easily use some commands in all containers.
+
+    Screwdriver should have a single interface for executing a versioned command
+    (via remote binary, docker image, or habitat package) during a build.
+
+    Background:
+        Given an existing pipeline with <image> image and a command:
+            | image         | namespace   | command        | version |
+            | golang:latest | sd-cmd-test | sample-command | 1.0.0   |
+
+    Scenario Outline: Execute a command of habitat format
+        Given command specifications as mode: <mode>, package: <package>, binary: <binary>
+        When the command package mode is <mode>
```

Comment:
probably can get rid of one of the `mode`s since it's on line 16 and 17

Reply:
Thanks, I removed this line.